README.md

```diff
@@ -87,27 +84,21 @@ To set up a local development-environment:
 ### On macOS
 
 #### Install dependencies
+- Install Git: <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git> > Installing on macOS (section)
+- Clone the Git repository: `git clone https://github.com/global-121/121-platform.git`
 - Install Node.js  
   To install the required version of Node.js and to prevent conflicts between projects or components using other versions of Node.js, use [NVM - Node Version Manager](http://nvm.sh/).  
   After installing NVM run:
 
       nvm install && nvm install-latest-npm
-
-- Install Git  
-  <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git> > Installing on macOS (section)
+- `cd` to the directory of the cloned repository above and execute `nvm install`. In this way nvm "sees" which version of NodeJS needs to be installed.
```

Comment:
See previous comment:

```
  After installing NVM run, from the root of this repository:  
  `nvm install && nvm install-latest-npm`
  So that the required version of Node.js and `npm` get installed.
```